text-generation-inference/server/text_generation_server/jetstream_pt_support/token_selector.py

```diff
@@ -55,6 +55,8 @@ def __init__(
         self.eos_token_ids = eos_token_ids
         self.pad_token_id = pad_token_id
         self.logits_warper = logits_warper
+        # Seed needs to fit a 64-bit integer, so we modulo it in case is bigger (that can happen!)
+        seed = seed % jnp.iinfo(jnp.int64).max
```

Comment:
Can't we just store it as int64? Modulo op sounds very expensive

Reply:
The seed comes from the TGI router and sometimes it comes as a crazy big number (bigger than int64). numpy and torch seem to silently do the operation, but `jax` requires the seed to be an int64 and it raises an error if the number is higher. Let me know if you think about a "cheaper" alternative.